proc_CPMG.py

```diff
@@ -1,7 +1,8 @@
 from pyspecdata import *
-from scipy.optimize import leastsq,minimize,basinhopping,nnls
+from scipy.optimize import basinhopping
 from proc_scripts import postproc_dict 
 fl = figlist_var()
+logger = init_logging('info')
 
 for searchstr, exp_type, nodename, postproc, label_str in [
         #('200221_CPMG_TEMPOLgel_3p0_1','test_equip','signal','CPMG','deadtime=5'),
```

Comment:
this postproc name "CPMG" is not unique enough -- needs to be something like `spincore_CPMG_v1` or somesuch

Reply:
again, no changes have been pushed (notice lack of "outdated")

Reply:
right because this section of code (from scipy and logger = ...) is up to data the changes it shows in green are accurate and there is no need to update these specific lines

Reply:
yes, but that's not what the comment is about -- it's about the name "CPMG" in the commented out code